shaptools/hana.py

```diff
@@ -195,14 +197,20 @@ def install(cls, software_path, conf_file, root_user, password, remote_host=None
             conf_file (str): Path to the configuration file
             root_user (str): Root user name
             password (str): Root user password
+            hdb_pwd_file (str, opt): Path to the XML password file
             remote_host (str, opt): Remote host where the command will be executed
         """
         # TODO: mount partition if needed
         # TODO: do some integrity check stuff
         platform_folder = cls.get_platform()
         executable = cls.INSTALL_EXEC.format(software_path=software_path, platform=platform_folder)
-        cmd = '{executable} -b --configfile={conf_file}'.format(
-            executable=executable, conf_file=conf_file)
+        if hdb_pwd_file:
+            cmd = ('cat {hdb_pwd_file} | {executable} -b '
```

Comment:
I would remove the parenthesis and add just `\` at the end of the first line. Besides that it looks good!